servers/rendering/shader_compiler.cpp

```diff
@@ -1183,9 +1183,10 @@ String ShaderCompiler::_dump_node_code(const SL::Node *p_node, int p_level, Gene
 
 					code += "(";
 
-					// if normal roughness texture is used, we will add logic to automatically switch between
+					// if color backbuffer, depth backbuffer or normal roughness texture is used,
+					// we will add logic to automatically switch between
 					// sampler2D and sampler2D array and vec2 UV and vec3 UV.
-					bool normal_roughness_texture_used = false;
+					bool multiview_used = false;
```

Comment:
```suggestion
					bool multiview_check_needed = false;
```

I don't think ``multiview_used`` is accurate for what the boolean is actually for

Reply:
good suggestion, I wasn't sure about the name either